package.json

```diff
@@ -10,7 +10,7 @@
 		"node": ">=10"
 	},
 	"scripts": {
-		"test": "xo && tsc --noEmit && nyc ava",
+		"test": "xo && tsc --noEmit && nyc --reporter=html --reporter=text ava",
```

Comment:
Very useful to debug later. Generates both HTML output and text output as usual.